os/include/tinyara/pm/pm.h

```diff
@@ -199,6 +199,18 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
+/* This enumeration provides all power management related wakeup source code. */
+
+typedef enum {
+	PM_WAKEUP_BLE,           /* BLE Interrupts */
+	PM_WAKEUP_WIFI,          /* Wifi Interrupts */
+	PM_WAKEUP_LOG_UART,      /* LOG UART Interrupts */
+	PM_WAKEUP_UART,          /* UART Interrupts */
+	PM_WAKEUP_GPIO,          /* Audio Interrupts */
+	PM_WAKEUP_HW_TIMER,      /* Timer Expiration */
+	PM_WAKEUP_SRC_COUNT,
```

Comment:
is it nessesary value?

Reply:
It's required in PM Metrics

Reply:
is PM_WAKEUP_UNKNOWN count which is needed in PM Metrics?
please use limit value only one

Reply:
@edwakuwaku Please refer change in PM_WAKEUP_UNKNOWN, we set it to 0 instead of 0xff

Reply:
Is this finalized version?